packages/snaps-sdk/src/jsx/components/form/AccountSelector.test.tsx

```diff
@@ -0,0 +1,106 @@
+import { AccountSelector } from './AccountSelector';
+
+describe('AccountSelector', () => {
+  it('returns an account selector element without filter props', () => {
+    const result = <AccountSelector name="account" />;
+
+    expect(result).toStrictEqual({
+      type: 'AccountSelector',
+      props: {
+        name: 'account',
+      },
+      key: null,
+    });
+  });
+
+  it('returns an account selector element with the chainIds filter prop', () => {
+    const result = (
+      <AccountSelector
+        name="account"
+        chainIds={['bip122:000000000019d6689c085ae165831e93']}
+      />
+    );
+
+    expect(result).toStrictEqual({
+      type: 'AccountSelector',
+      props: {
+        name: 'account',
+        chainIds: ['bip122:000000000019d6689c085ae165831e93'],
+      },
+      key: null,
+    });
+  });
+
+  it('returns an account selector element with the hideExternalAccounts filter prop', () => {
+    const result = (
+      <AccountSelector name="account" hideExternalAccounts={true} />
+    );
+
+    expect(result).toStrictEqual({
+      type: 'AccountSelector',
+      props: {
+        name: 'account',
+        hideExternalAccounts: true,
+      },
+      key: null,
+    });
+  });
+
+  it('returns an account selector element with the switchGlobalAccount filter prop', () => {
+    const result = (
+      <AccountSelector name="account" switchSelectedAccount={true} />
```

Comment:
Test title doesn't match the actual prop. I think `switchGlobalAccount` makes more sense though?